instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -603,6 +608,14 @@ async def __call__(self, scope, receive, send):
                     self.server_response_size_histogram.record(
                         self.content_length_header, duration_attrs
                     )
+                request_size = asgi_getter.get(scope, "content-length")
+                if request_size:
+                    try:
+                        self.server_request_size_histogram.record(
```

Comment:
Consider wrapping with try and except only the int cast, so the call to self.server_request_size_histogram.record will be more readable and clean

Reply:
@shalevr Thanks for the comment.
Much better now.
Let me know what you think.